requirements-dev.txt

```diff
@@ -10,4 +10,7 @@ autodocsumm>=0.2.10
 nbsphinx>=0.8.12
 sphinx-rtd-theme>=1.1.1
 # urllib3 v2 requires OpenSSL v1.1.1+, but Vercel uses v1.0.2
-urllib3<2.0
\ No newline at end of file
+urllib3<2.0
+jinja2
+ipywidgets
+tqdm
```

Comment:
why do we need these new ones? I don't see where they're being used

Reply:
As a rule of thumb let's try to only add requirements if it's absolutely necessary. So e.g. if `tqdm` is not actually needed by us then don't include it.